src/Tempest/Database/DatabaseDriverInitializer.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Database;
+
+use Tempest\Container\Container;
+use Tempest\Container\Initializer;
+use Tempest\Container\Singleton;
+
+#[Singleton]
+final class DatabaseDriverInitializer implements Initializer
```

Comment:
added wiring for the database driver, so I don't have to register it manually

Reply:
FYI, you'll need to move the `#[Singleton]` attribute to the `initialize` method instead of the class. This was a change I made this morning :)